source/geod24/Transceiver.d

```diff
@@ -0,0 +1,352 @@
+/*******************************************************************************
+
+    This is an abstract of the transmission and reception module of the message.
+    Send and receive messages using the Channel in `geod24.concurrency`.
+    Fiber
+
+    It is the Scheduler that allows the channel to connect the fiber organically.
+
+*******************************************************************************/
+
+module geod24.Transceiver;
+
+import geod24.concurrency;
+
+import core.time;
+
+
+/// Data sent by the caller
+public struct Request
```

Comment:
The various commands (`FilterAPI`, `Status`, `Response`, `Request`, `TimeCommand`, `ShutdownCommand`) should live in `LocalRest`, nowhere else. They are internal details.

If you need a [tagged union](https://en.wikipedia.org/wiki/Tagged_union) for channel communication, you can use [Minivariant](https://github.com/Geod24/minivariant). You don't even need to import it, it is trivial to have a variant type in D, but it does provide some nice-to-have tools. There's also other types that do that, such as [sumtypes](https://github.com/pbackus/sumtype).